Jaman-dedy/shortly_mobile_app
# implement the home component

#### Description of changes made

> This PR adds the home component to the codebase.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the `git@github.com:Jaman-dedy/shortly_mobile_app.git`
- Check out this branch by `ft-static-component`
- Run `yarn install` to install the project dependencies
- Run `yarn android`

#### Checklist


#### GitHub issue your changes fix

#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| Before these changes | After the changes |
|                      |     
<img width="341" alt="image" src="https://user-images.githubusercontent.com/46047244/170527013-274f5234-942e-4116-88cb-6969020e09c4.png">
              |
